docs/en_US/_templates/index.html

```diff
@@ -0,0 +1,506 @@
+{% extends "!layout.html" %}
+
+{% set title = "Welcome To Neural Network Intelligence !!!"%}
+
+{% block document %}
+
```

Comment:
Add a link to Chinese? Though users can select zh in readthedocs menu, but it's not obvious.